src/app/Routes.tsx

```diff
@@ -191,10 +199,18 @@ const Routes = (): JSX.Element => (
       <Route
         element={
           <ErrorBoundary>
-            <KVM />
+            <ImageList />
+          </ErrorBoundary>
+        }
+        path={`${urls.images.index}/*`}
+      />
+      <Route
+        element={
+          <ErrorBoundary>
+            <MachineDetails />
           </ErrorBoundary>
         }
-        path={`${urls.kvm.index}/*`}
+        path={`${urls.machines.machine.index(null)}/*`}
```

Comment:
Looks like a mistake from resolving merge conflicts, could you please remove.

These have been moved to use the new page wrapper, see the lines above: https://github.com/canonical/maas-ui/pull/4989/files#diff-cf93f5edcf06a2c153726ff6ffd1514499db8618fe0592bbee81f0fa5a8887faR151